aula/apps/extPreinscripcio/models.py

```diff
@@ -49,7 +49,13 @@ class Meta:
         
     def __str__(self):
         return str(self.cognoms) + ', ' + str(self.nom) + ': ' + str(self.codiestudis) + ' ' + str(self.curs)
-
+    
+    def getCurs(self):
+        try:
+            return Curs.objects.get(nivell__nom_nivell=self.codiestudis, nom_curs=self.curs)
+        except Exception:
```

Comment:
Millor especifiquem l'excepció (per no amagar excepcions no relacionades), podem fer servir el `ObjectDoesNotExist` o bé:

```suggestion
        except Curs.DoesNotExist:
```